tests/migrations/test_commands.py

```diff
@@ -510,6 +510,26 @@ def test_files_content(self):
                     self.assertIn('\\xda\\xd1\\xcd\\xa2\\xd3\\xd0\\xc9', content)  # title.verbose_name
                     self.assertIn('\\u201c\\xd0j\\xe1\\xf1g\\xf3\\u201d', content)  # title.default
 
+    def test_makemigrations_order(self):
+        def create_migration():
+            if hasattr(importlib, 'invalidate_caches'):
```

Comment:
Yes. And it took some time to discover what was happening. It leads to weird import errors.